src/transaction/log_prior_recv.cpp

```diff
@@ -28,12 +28,14 @@ namespace cublog
   prior_recver::prior_recver (log_prior_lsa_info &prior_lsa_info)
     : m_prior_lsa_info (prior_lsa_info)
   {
-    start_thread ();
```

Comment:
`prior_recver` is also used by PTS.
I am not mistaken, removing this from here breaks PTS since for PTS this thread is not started anywhere.
- the - now public - `prior_recver::start_thread` function must be called in `server_type` for PTS
- or, feel free to create a new `log_global::initialize_and_start_log_prior_receiver` that does both and call that one for PTS in `server_type`

Reply:
Definitely. I should've considered it.

Sorry, I've tended to miss PTS's basic functionality. To prevent it, I'll create and conduct a small test sets for it like starting/stopping servers and a basic replication behavior.

Reply:
done: [8bfe418](https://github.com/CUBRID/cubrid/pull/4721/commits/8bfe418493ce3d8d226b6f1359ca3ca7ae4b4fa0)